### DISCUSSION TITLE
Page Dynamic Table - p-actions-right

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#1133] [Tire suas dúvidas] [answered]
Não consegui colocar o "actions" do lado direito, o portal também ainda não tem o "p-actions-right".

**Comment by alinelariguet** [comment, marked as answer]
Boa tarde @wsteixeira 
Essa propriedade ainda não foi repassada para o componente po-page-dynamic-table, está disponível ainda somente no componente po-table. Você pode criar uma issue a partir dessa discussão e colocaremos essa feature no nosso backlog, ou então, você também pode contribuir com essa funcionalidade :)
Atenciosamente

**Reply by wsteixeira** [reply]
Boa tarde @alinelariguet 

     Já criei um issue sobre este assunto e também já estou preparando a contribuição.
